### PR TITLE
Add UpdateItem Func

### PR DESCRIPTION
**Pull request by schreibe72**
I impemented a function to update a item.

### CODE REVIEW

File: keychain.go [hunk 1]
```diff
@@ -237,6 +237,19 @@ func AddItem(item Item) error {
...
 
+// UpdateItem updates a Item
+func UpdateItem(item Item) error {
+	cfDict, err := ConvertMapToCFDictionary(item.attr)
+	if err != nil {
+		return err
+	}
+	defer Release(C.CFTypeRef(cfDict))
+
+	errCode := C.SecItemUpdate(cfDict, cfDict)
```
Comment (gabriel) [inline keychain.go]: Is it awkward to pass in the same dict as both args? or is that the normal way to update?

**Comment by gabriel** [comment]
Possible to add a test?

**Comment by schreibe72** [comment]
Sorry, i was quite to fast with making a pull request. But now i fixed UpdateItem and wrote a test for it. I only used this function on MacOS and not IOS. So i don't know if this function works on IOS.

File: keychain.go [hunk 2]
```diff
@@ -237,6 +237,23 @@ func AddItem(item Item) error {
 	return err
 }
 
+// UpdateItem updates the queryItem with the Parameters from updateItem
```
Comment (gabriel) [inline keychain.go]: Parameters -> parameters

**Pull request merged**